opal/mca/btl/openib/help-mpi-btl-openib.txt

```diff
@@ -250,6 +250,12 @@ this run.
 
 If the "requested type" is "<any>", this usually means that *no*
 OpenFabrics devices were found within the requested bus distance.
+
+Note starting with Open MPI 4.0, only iWarp and RoCE devices are considered
+for selection by default.  Either set the btl_openib_allow_ib MCA
+parameter to "true" or set the btl_openib_device_type MCA parameter to 
+"ib" or "infiniband".
+
```

Comment:
Watch extra blank lines in the help file (because they will be displayed to the user).